code/game/objects/items/tools/maintenance_tools.dm

```diff
@@ -654,14 +654,19 @@ Welding backpack
 /obj/item/tool/weldpack/afterattack(obj/O as obj, mob/user as mob, proximity)
```

Comment:
Do the same as the engineer packs, replace `obj/O as obj` with `obj/target`

Reply:
i did i hope it work